db/schema.rb

```diff
@@ -1184,7 +1185,7 @@
   end
 
   create_table "schedule_activities", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
-    t.bigint "venue_room_id", null: false
+    t.bigint "venue_room_id"
```

Comment:
Unrelated change (but you have merge conflicts anyways...)

Reply:
Fixed